Apps/WebClient/src/ClientApp/app/boot.ts

```diff
@@ -6,7 +6,7 @@ import VueRouter from "vue-router";
 
 import "bootstrap-vue/dist/bootstrap-vue.css";
 import "@/assets/scss/bcgov/bootstrap-theme.scss";
-import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { FontAwesomeIcon, FontAwesomeLayers } from "@fortawesome/vue-fontawesome";

```

Comment:
Didnt see the layers being used on the code, but probably I missed it since I havent use it before. How does it work?

Reply:
Tried it out and it didn't do what I was hoping for - just forgot to remove the import